composer.json

```diff
@@ -14,6 +14,6 @@
         }
     },
     "require": {
-        "wikibase/data-model": "^7.3"
+        "wikibase/data-model": "^9|^8|^7"
```

Comment:
This could also be `"^9.5.1|^8.0.0|^7.3.0"` instead; I don’t think it makes much of a difference, but let me know if you’d prefer that.

Reply:
Changed it to that for consistency with adrianheine/mw-wikibase-forms#1.